timeago/lib/src/messages/ar_messages.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:timeago/src/messages/lookupmessages.dart';
+
+class EnMessages implements LookupMessages {
```

Comment:
these classes should be named after the locale names, in this case the name should be `ArMessages` and `ArShortMessages`

Reply:
@andresaraujo oops!
Okay I've edited them